tests/cli/test_cli.py

```diff
@@ -0,0 +1,22 @@
+def test_cli_start(run):
```

Comment:
@wochinge This test should cover the travis test, correct?

Reply:
yep, should we run that multiple times to make the built more robust?